src/index.js

```diff
@@ -8,6 +8,7 @@ angular
         $httpBackend.whenGET('/api/threads').respond(mockBackend.getThreads);
         $httpBackend.whenGET('/api/updates').respond(mockBackend.getUpdates);
         $httpBackend.whenGET('/api/owner').respond(mockBackend.getOwner);
+        $httpBackend.whenGET('/api/screenTypes').respond(mockBackend.getScreenTypes);
```

Comment:
Why is this an API call? Screen types are static, right? They don't depend on user actions or account. You could save yourself the trouble of making an API call to fetch screen types, as they never vary.